<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewGenerixDerivation2Page</class>
 <widget class="QWizardPage" name="NewGenerixDerivation2Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <property name="title">
   <string>Define the source and destination path</string>
  </property>
  <property name="subTitle">
   <string>In derivation mode you must define the path where the new file are stored. You can define the source path, as the start of the tree to copy.</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="m_derivationLabel">
     <property name="text">
      <string>&amp;Derivation path :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DirectoryEditWidget" name="m_derivationPath" native="true"/>
   </item>
   <item>
    <widget class="QLabel" name="m_sourceLabel">
     <property name="text">
      <string>&amp;Source path :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="DirectoryEditWidget" name="m_sourcePath" native="true"/>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_2">
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectoryEditWidget</class>
   <extends>QWidget</extends>
   <header>directoryedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
